packages/management/featuregates/bundle/config/upstream/tanzu-featuregates-manager.yaml

```diff
@@ -46,6 +46,19 @@ spec:
             secretName: tanzu-featuregates-webhook-server-cert
       serviceAccount: tanzu-featuregates-manager-sa
       terminationGracePeriodSeconds: 10
+      #@ if hasattr(data.values, 'deployment') and data.values.deployment.hostNetwork:
+      #@overlay/match missing_ok=True
+      hostNetwork: true
```

Comment:
Why do we need to run this pod with `hostNetwork=true`? I can't find an explanation in the linked issue.